
Due to disagreement how to solve this correctly we keep this private patch
in order to use the radeon driver on iBooks without the UseFBDev hack.

  - Rene Rebe - T2 project

diff -u -r1.19 radeon_driver.c
--- xc/programs/Xserver/hw/xfree86/drivers/ati/radeon_driver.c	25 Aug 2004 00:30:41 -0000	1.19
+++ xc/programs/Xserver/hw/xfree86/drivers/ati/radeon_driver.c	28 Aug 2004 19:28:47 -0000
@@ -6755,6 +6755,11 @@
 
     save->ppll_ref_div   = pll->reference_div;
     save->ppll_div_3     = (save->feedback_div | (post_div->bitvalue << 16));
+
+#if defined(__powerpc__)
+    save->ppll_div_3     = 0x000600ad; // hack for iBooks, needs some review ...  -ReneR
+#endif
+
     save->htotal_cntl    = 0;
 }
 
